src/dosbox.cpp

```diff
@@ -362,7 +364,7 @@ static void DOSBOX_RealInit(Section * sec) {
 	Section_prop * section=static_cast<Section_prop *>(sec);
 	/* Initialize some dosbox internals */
 
-	use_delay_precise = CanDelayPrecise();
+	delay_fn = CanDelayPrecise() ? DelayPrecise : Delay;
```

Comment:
Just a question - the other parts of the code-base that still use `Delay()`, we don't want them using `delay_fn` because they don't require strict timing?

As a casual ready, `delay_fn` and `Delay` feel a bit ambiguous.. it's easy enough to see that one is a function pointer, while the other is the function - but readers might wonder why `delay_fn` isn't used throughout?

Maybe renaming `Delay` to `DelayCoarse` or `DelayImprecise`, and then use that in the other parts of the code - it would tell reads "we are deliberately using the looser delay".

Reply:
I was thinking we are only overriding the old Delay() behavior people expect for this file, specifically only in `increaseticks()` (again going for that least impact when pulling from SVN, so Delay() anywhere is still Delay())